selftests/functional/test_interrupt.py

```diff
@@ -121,6 +121,9 @@ def wait_until_no_badtest():
         # Make sure the Killing test subprocess message did appear
         self.assertIn('Killing test subprocess', proc.get_output())
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
You can't filter by tags when running it as unittest, so NO to your first note and yes please enable it in Travis (until it starts breaking...)